common/constants.ts

```diff
@@ -204,7 +204,109 @@ export enum WAZUH_MODULES_ID{
   TSC = 'tsc',
   CIS_CAT = 'ciscat',
   VIRUSTOTAL = 'virustotal',
-  GDPR = 'gdpr'
+  GDPR = 'gdpr',
+}
+
+export enum WAZUH_MODULES_CY_TEST_ID {
+  SECURITY_EVENTS = 'menu-modules-general-link',
+  INTEGRITY_MONITORING = 'menu-modules-fim-link',
+  AMAZON_WEB_SERVICES = 'menu-modules-aws-link',
+  GOOGLE_CLOUD_PLATFORM = 'menu-modules-gcp-link',
+  POLICY_MONITORING = 'menu-modules-pm-link',
+  SECURITY_CONFIGURATION_ASSESSMENT = 'menu-modules-sca-link',
+  AUDITING = 'menu-modules-audit-link',
+  OPEN_SCAP = 'menu-modules-oscap-link',
+  VULNERABILITIES = 'menu-modules-vuls-link',
+  OSQUERY = 'menu-modules-osquery-link',
+  DOCKER = 'menu-modules-docker-link',
+  MITRE_ATTACK = 'menu-modules-mitre-link',
+  PCI_DSS = 'menu-modules-pci-link',
+  HIPAA = 'menu-modules-hipaa-link',
+  NIST_800_53 = 'menu-modules-nist-link',
+  TSC = 'menu-modules-tsc-link',
+  CIS_CAT = 'menu-modules-ciscat-link',
+  VIRUSTOTAL = 'menu-modules-virustotal-link',
+  GDPR = 'menu-modules-gdpr-link',
+}
+
+export enum WAZUH_MANAGEMENT_ID {
```

Comment:
the name of the constants are not easy to associate with his purpose, they are very generics

Reply:
Good observation there I check it.